### PR TITLE
Fixes `@torch.no_grad()` usage

### PR DESCRIPTION
**Pull request by fracapuano**
## What this does
Applies a very minor fix on top of all policies running inference, to make sure CUDA memory consumption is limited at inference time (ensured via the `@torch.no_grad()` decorator---wrongly added as `@torch.no_grad` and now fixed).

### CODE REVIEW

**Review by imstevenpmwork — DISMISSED** [review]
LGTM, but can you also add in `src/lerobot/policies/normalize.py` to the `forward` methods?

**Review by imstevenpmwork — APPROVED** [review]
